_data/companies.yml

```diff
@@ -0,0 +1,803 @@
+  location: Japan
+  foundation: 2014
+  urls:
+    - https://miraitranslate.com/
+    - https://www.crunchbase.com/organization/mirai-translate-inc
+
+- name: ModelFront
+  id: modelfront
+  type: [ mt-aggregator ]
+  location: United States
+  foundation: 2019
+  urls:
+    - https://www.modelfront.com/
+    - https://www.crunchbase.com/organization/modelfront
+
+- name: Moravia
+  id: moravia
+  type: [ lsp ]
+  location: Czech Republic
+  foundation: 2014
+  urls:
+    - http://www.moravia.com/
+    - https://www.crunchbase.com/organization/moravia
+  acquired_by:
+    company: RWS
+
+- name: Naver Cloud
+  id: naver-cloud
+  type: [ technology-provider ]
+  location: South Korea
+  foundation: 2009
+  urls:
+    - https://www.navercloudcorp.com/
+    - https://www.crunchbase.com/organization/naver-cloud
+
+- name: National Institute of Information and Communications Technology
+  id: nict
+  type: [ mt-provider ]
+  location: Japan
+  foundation: 1896
+  urls:
+    - https://www.nict.go.jp/en/index.html
+    - https://www.crunchbase.com/organization/national-institute-of-information-and-communications-technology-japan
+  note: NICT products include a text machine translation TexTra and a voice machine translation, VoiceTra.
+
+- name: NetEase
+  id: netease
+  type: [ technology-provider ]
+  location: China
+  foundation: 1997
+  urls:
+    - https://www.neteasegames.com
+    - https://www.crunchbase.com/organization/netease-com
+
+- name: NpatMT
+  id: npatmt
+  type: [ mt-provider ]
+  location: Japan
+  foundation: 2015
+  urls:
+    - https://www.npat.co.jp/
+
+- name: Omniscien Technologies
+  alternative_names: [ Asia Online ]
+  id: omniscien-technologies
+  type: [ mt-provider ]
+  location: The Netherlands
+  foundation: 2006
+  urls:
+    - https://www.omniscien.com
+    - https://www.crunchbase.com/organization/omniscien-technologies-trading
+
+- name: Oneliner
+  id: oneliner
+  type: [ lsp, mt-provider ]
+  location: Belgium
+  foundation: 1999
+  urls:
+    - https://oneliner.be/en/home.html
+    - https://www.crunchbase.com/organization/oneliner
+
+- name: Pangeanic
+  id: pangeanic
+  type: [ mt-provider, lsp ]
+  location: Spain
+  foundation: 2000
+  urls:
+    - https://www.pangeanic.com/
+    - https://www.crunchbase.com/organization/pangeanic
+
+- name: Panlingua
+  id: panlingua
+  type: [ mt-provider ]
+  location: India
+  foundation: 2017
+  urls:
+    - http://panlingua.co.in/
+    - https://github.com/panlingua
+
+- name: Precision Translation Tools
+  id: precision-translation-tools
+  type: [ technology-provider ]
+  active: false
+  urls:
+    - http://www.precisiontranslationtools.com/
+
+- name: Prompsit Language Technology
+  id: prompsit
+  type: [ technology-provider ]
+  location: Spain
+  foundation: 2006
+  urls:
+    - https://prompsit.com/
+    - https://www.crunchbase.com/organization/prompsit-language-engineering
+
+- name: PROMT
+  id: promt
+  type: [ mt-provider ]
+  location: United States
```

Comment:
Are we sure it's not Russia?

Reply:
I didn't realize both Russia and the US were mentioned in its Crunchbase entry. Fixed.